docs/versioned/api/java-api-quickstart.md

```diff
@@ -39,7 +39,7 @@ You can initialize a Hive catalog with a name and some properties.
 ```java
 import org.apache.iceberg.hive.HiveCatalog;
 
-Catalog catalog = new HiveCatalog();
+HiveCatalog catalog = new HiveCatalog();
 catalog.setConf(spark.sparkContext().hadoopConfiguration());  // Configure using Spark's Hadoop configuration
```

Comment:
`setConf` is not a method of `Catalog`, it is a method of `org.apache.hadoop.conf.Configurable`, which `HiveCatalog` also implements.